README.md

```diff
@@ -1,24 +1,213 @@
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
+  <br/>
 
-* Database creation
+  <h3><b>Budget App - RoR</b></h3>
 
-* Database initialization
+</div>
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📗 Table of Contents
 
-* Deployment instructions
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Ruby on Rails ](#ruby-on-rails-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**The Budget App** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Ruby on Rails <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Spash Screen**
+- **Sign up and log in pages**
+- **Home page (catagories page)**
+- **Transactions page**
+- **"Add a new category" page**
+- **"Add a new transaction" page**
+- **Unit & Integration Testing with RSpec & Capybara**
+- **Aunthentication with Devise**
+- **Database management with Postgres**
+- **ORM with ActiveRecord**
+- **Deployment**
+- **Project Demo with Loom**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> View Video Demo [here](https://www.loom.com/share/0ff174560ebd4edfa4ee68f389f36836)
```

Comment:
- [ ] As requested by the previous reviewer, **please run the project locally**, and **record a video presentation** using loom. In the video you will record, **please present the functionality of your web app** and also **present the piece of code and how it works.** Easy and understandable, right? 😃 
After recording the video, you will need to replace the current video link in your README with the updated one 😃 
Good luck 🏃